projects/common-platform-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/integrations/delius/entity/Person.kt

```diff
@@ -99,6 +99,18 @@ class Person(
     @Column(name = "DEFENDANT_ID")
```

Comment:
Minor nitpick - could we make these lowercase so they're consistent with the other fields on the entity?